src/engine/sync/enginesync.cpp

```diff
@@ -56,9 +56,25 @@ void EngineSync::requestSyncMode(Syncable* pSyncable, SyncMode mode) {
         [[fallthrough]];
     }
     case SyncMode::LeaderSoft: {
-        if (pSyncable->getBaseBpm().isValid() &&
-                pSyncable->isPlaying() &&
-                pSyncable->isAudible()) {
+        if (pSyncable->getBaseBpm().isValid()) {
+            if (!pSyncable->isPlaying()) {
+                bool anyPlaying = false;
+                // Only allow a non-playing deck to be leader if no deck is playing
```

Comment:
Do we have a use case for this? 
* Load two decks. 
* sync both. 
* select a soft-leader
* play the soft leader -> it is picked anyway 
* play the follower -> it becomes the new leader

So it looks like users choice is ignored anyway.

Reply:
> So it looks like users choice is ignored anyway.

I think I can fix that be preferring the current leader if there are multiple equivalent choices